float64[3] velodynepoint
float64[3] velodynenormal
float64[3] camerapoint
float64[3] cameranormal
float64[3] velodynecorner
float64 pixeldata
